### PR TITLE
Remove bash prompt chars

### PR DESCRIPTION
**Pull request by ahgittin**
inspired by #97 (and replacing it) this uses the proper multi-line bash char, and ensures these chars are removed on copy-paste

uses a nice JS `oncopy` trick mentioned in the comments on that PR

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -65,32 +63,70 @@
...
-      // at the very start, or immediately following any newline). Correctly handles continuation
-      // lines, where a leading '$' or '%' is *not* a prompt character.
-      txt = txt.replace(/(^|[^\\]\n)[$%] /g, "$1");
-    }
+    var target0 = $(this).next();
+    var target = target0.clone();
+    target.find('.nocopy').remove();
+    var txt = target.text();
     clipboard.on( 'dataRequested', function (client, args) {
+      handleHideCopy( target0.closest('div.highlight') );  //not necessary but nicer feedback
```
Comment (neykov) [inline _layouts/base.html]: This won't be called if zeroclipboard is not active (no flash installed for example). Could use [`user-select`](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select) for a similar effect.

Comment (ahgittin) [inline _layouts/base.html]: yeah the `oncopy` handler solves the no-flash case.  would be nice if we didn't even show the clipboard on no-flash but didn't do that.

i have added `user-select` in a follow-up PR, but note it doesn't always do what one might expect, so this code is worth having in addition

Comment (neykov) [inline _layouts/base.html]: Oh it's attached to the `oncopy` event - missed that. Should be good enough then.

**Comment by neykov** [comment]
LGTM

**Pull request merged**